docs/src/CHANGELOG.md

```diff
@@ -5,6 +5,39 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
  
+## [Unreleased]
+
+This release added the ability to regrid data stored on a UGRID mesh.
+
+### Added
+
+- [PR#51](https://github.com/SciTools-incubator/iris-esmf-regrid/pull/51)
+  [PR#55](https://github.com/SciTools-incubator/iris-esmf-regrid/pull/55)
+  [PR#96](https://github.com/SciTools-incubator/iris-esmf-regrid/pull/96)
+  Added the unstructured regridders `GridToMeshESMFRegridder` and
+  `MeshToGridESMFRegridder`.
+  [@stephenworsley](https://github.com/stephenworsley)
+- [PR#132](https://github.com/SciTools-incubator/iris-esmf-regrid/pull/132)
+  Added functions for saving of the unstructured regridders.
+  [@stephenworsley](https://github.com/stephenworsley)
+- [PR#132](https://github.com/SciTools-incubator/iris-esmf-regrid/pull/132)
+  improved generation of benchmark data to allow caching of generated data.
+  [@trexfeathers](https://github.com/trexfeathers)
```

Comment:
I wanted to include contributions from other people where possible. I don't know if theres a way in the changelog conventions to include credit to people for doing review work, because I feel like there's a lot of work there that you wouldn't see by just listing who authored the PRs.

Reply:
>I wanted to include contributions from other people where possible.

Yes, good point. There's no actual convention in [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) about referencing commits/PR's/etc. It even says:

>Changelogs are _for humans_, not machines.

So I suggest including a couple of bullet points acknowledging @jamesp for extensive reviewing and @trexfeathers for benchmarking collaboration. Something like that?